summary_workflows/quantification/quantification_dockers/q2_metrics/compute_metrics.py

```diff
@@ -54,7 +54,7 @@ def compute_metrics(input_participant, gold_standards_dir, challenge_types, part
         # get metrics dataset
 #        metrics_data = pandas.read_csv(os.path.join(gold_standards_dir, challenge + ".txt"),
 #                                       comment="#", header=0, sep='\t',index_col=0)
-        metrics_data=os.path.join(gold_standards_dir, "siControl_R2.MACEseq.mm10.bed") ##PLEASE CHANGE THIS LATER
+        metrics_data=os.path.join(gold_standards_dir, challenge + ".bed") ##NO LONGER HARD-CODED
 
         # metric on the number of matched sites
         window = 15
```

Comment:
Why is the window size hardcoded? Do we want to change that?

Reply:
I have added `window` as an argument (default: 15) to optionally pass in different window size

Reply:
Thanks @yuukiiwa , I added a commit b5773862f9c2bd8301da76c14af76bb1db89ecbe such that the window parameter is passed through.